### PR TITLE
adds more informative notes to submission-ui

### PR DESCRIPTION
**Pull request by SBBCornell**
This PR adds more informative notes to submission-ui to bring to users attention some common causes of publication delay. These issues will also be addresses through programmatic changes, the text changes are a first pass. The added text is:

- a note that filenames are case sensitive on the upload page.
- a note on the processing page about oversize files. 
- and on the upload and to processing pages, a note about missing references.

This PR also adds a close button to the success message on the processing page. It takes up quite a lot of space and there is more to read below in the log.

Screenshots:

Screen Shot 2020-02-21 at 8 43 14 AM

Screen Shot 2020-02-21 at 8 45 05 AM

### CODE REVIEW

File: submit/templates/submit/file_process.html [hunk 1]
```diff
@@ -38,8 +38,10 @@
       <p>What to do next:</p>
       <ul>
         <li>review the log warnings and errors</li>
-        <li>correct any missing file problems</li>
+        <li>correct any missing file problems.
+        <li>make sure links to included files match the names of the files exactly (it is case sensitive)</li>
         <li>edit your files if needed</li>
+        <li>double check captions, references and citations.</li>
```
Comment (mhl10) [inline submit/templates/submit/file_process.html]: I recommend putting references first"

```suggestion
        <li>verify your references, citations and captions.</li>
```

File: submit/templates/submit/file_process.html [hunk 2]
```diff
@@ -61,28 +63,30 @@
...
         Processing Successful
       </p>
     </div>
     <div class="message-body">
-      <p class="has-text-weight-bold">Be sure to check the compiled preview carefully for any undesired artifacts or errors. Reference numbers and captions are important to check.</p>
+      <p class="has-text-weight-bold">Be sure to check the compiled preview carefully for any undesired artifacts or errors.</p>
       <p>If you need to make corrections, return to the previous step and adjust your files, then reprocess your submission.</p>
+      <p>
+        <strong>Avoid common causes of delay:</strong>
+        Double check captions, references and citations.
```
Comment (mhl10) [inline submit/templates/submit/file_process.html:87]: This should be consistent with above suggestion, if you agree with it

**Review by mhl10 — APPROVED** [review]
This looks good and will be helpful. I made a minor suggestion about the references text.

File: submit/templates/submit/file_process.html (hunk 2 above)
Comment (SBBCornell) [reply submit/templates/submit/file_process.html:87]: Done!

File: submit/templates/submit/file_process.html [hunk 3]
```diff
@@ -38,8 +38,10 @@
       <p>What to do next:</p>
       <ul>
         <li>review the log warnings and errors</li>
-        <li>correct any missing file problems</li>
+        <li>correct any missing file problems.
+        <li>make sure links to included files match the names of the files exactly (it is case sensitive)</li>
```
Comment (bdc34) [inline submit/templates/submit/file_process.html:42]: The text about "files match the names of the files exactly" is not clear. 

I suspect that user testers would not be able to explain what that sentence if we asked them about it.

I suggest detecting that a "file not found error" in the processing log matches an existing file and then showing a specific error message.

If this is a stop gap solution until we can do the work to make a error check like that, please mark this comment as resolved.

Comment (SBBCornell) [reply submit/templates/submit/file_process.html:42]: Correct, these are quick edits we can make now as a stop-gap. They dont preclude solutions like enhancing checks and error messages that we'll continue to work on in future sprints.

Comment (bdc34) [reply submit/templates/submit/file_process.html:42]: :+1:

**Pull request merged**